luigi/worker.py

```diff
@@ -160,6 +160,7 @@ def run(self):
             # Need to have different random seeds if running in separate processes
             random.seed((os.getpid(), time.time()))
 
+        t0 = time.time() # Failed task start time
```

Comment:
This is unnecessary. The declaration of `t0` on line 179 is still in scope inside the `except` block.